tools/releaser/scripts/breaking-changes.sh

```diff
@@ -3,24 +3,21 @@ set -eu
 GOPATH=$(go env GOPATH)
 
 # Inputs:
-# GIT_BASE_REF - The base REF of the git repository (git rev-parse origin/main)
-# Usually "${{ github.event.pull_request.base.sha || github.event.merge_group.base_sha }}" 
 # TARGET_BREAKING_CHANGES_FILE - file to save breaking changes
 script_path=$(dirname "$0")
 
 echo "Installing go-apidiff"
 go install github.com/joelanford/go-apidiff@latest > /dev/null
 
-current_ref=$(git rev-parse main || echo)
-GIT_BASE_REF=${GIT_BASE_REF:-$current_ref}
+GIT_BASE_REF=$(git rev-parse HEAD^)
```

Comment:
compare last commit and previous commit

Reply:
Since we no longer use default configs any potential change in the github action will break this job.
I would rather to provide 2 ref for comparison from the github action job to avoid regressions. 

This way we can avoid regressions that are very hard to track. 
In this case I think it would be assumption in this script that we're going to have 3 commits.

Reply:
thanks, done